Explorer/Assets/Scripts/ECS/Unity/Materials/Tests/CleanUpMaterialsSystemShould.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Arch.Core;
+using ECS.LifeCycle.Components;
+using ECS.StreamableLoading.Components.Common;
+using ECS.TestSuite;
+using ECS.Unity.Materials.Components;
+using ECS.Unity.Materials.Systems;
+using NSubstitute;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace ECS.Unity.Materials.Tests
+{
+    public class CleanUpMaterialsSystemShould : UnitySystemTestBase<CleanUpMaterialsSystem>
+    {
+        private IMaterialsCache materialsCache;
+
+        private Entity e;
+        private Entity texPromise;
+
+        [SetUp]
+        public void SetUp()
+        {
+            system = new CleanUpMaterialsSystem(world, materialsCache = Substitute.For<IMaterialsCache>());
+
+            e = world.Create(new MaterialComponent { AlbedoTexPromise = world.Reference(texPromise = world.Create()) }, new DeleteEntityIntention());
+        }
+
+        [Test]
+        public void AbortLoadingIntentions()
+        {
+            ref MaterialComponent component = ref world.Get<MaterialComponent>(e);
+            component.Status = MaterialComponent.LifeCycle.LoadingInProgress;
+
+            system.Update(0);
+
+            Assert.That(world.Has<ForgetLoading>(texPromise), Is.True);
```

Comment:
To stay consistent in the code, should I follow the convention of using `Assert.That` instead of `Assert.IsNull`, `Assert.IsTrue`, etc?

Reply:
`Copilot` is to blame 😄 I don't see much difference, just use what you like